neural_compressor_ort/__init__.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
```

Reply:
Consider removing from all files. #!/usr/bin/env python Can be kept only for executable files.

Reply:
Ok